phsyn-t3/cod-u8776t_phsyn-t3.xml

```diff
@@ -57,881 +57,1053 @@
+            <lb ed="#T"/>dicebatur quod nullus. Ergo a primo: si Sortes non videt aliquem hominem <del rend="strikethrough">non</del>
+            <lb ed="#T"/> videntem se, Sortes non videt se. Et sic habetur alia pars
+            contradictio<lb ed="#T"/>nis. Ex qua cum altera expositiva prime sequitur reliqua pars
+            <lb ed="#T"/> contradictionis, sicut prius. Quare prima habet se contradictionem. Quod concedimus.</p>
+
+          <p xml:id="phsyn-t3-d1e375">
+            <lb ed="#T"/><pc type="#pilcrow"/>Item. Probo quod prima est vera, hec scilicet ‘Sortes videt tantum omnem hominem
+            <lb ed="#T"/>non videntem se’. Ponatur quod Virgilius et Plato et Cicero situ homines
+            <lb ed="#T"/>non videntes se tantum et omnes alii videant se; et ponatur quod Sortes videat tantum illos tres, quia hec positio possibilis est. Et
+            <lb ed="#T"/><seg type="correction"> tunc hec
+                <subst>
+                    <del>cum hoc</del>
+                    <add>tunc hec</add>
+                  </subst>
+                </seg> <!--another hand corrects the lesson by ammending the existing letters into new ones!-->
+            ‘omnis homo non videns se non det se’ convertitur cum hiis tribus singularibus
+            <lb ed="#T"/> ‘Virgilius non videt se’, ‘Plato non videt se’, ‘Cicero non videt se’. Et iste
+            <lb ed="#T"/> tres insimul sumpte convertuntur cum illa, quia sequitur omnis homo non
+            vi<lb ed="#T"/>dens se egit; ergo et iste et hic et ille legit et econverso. Sed Sortes
+            vi<lb ed="#T"/>det tantum illos tres. Ergo Sortes videt tantum omnem hominem non
+            <lb ed="#T"/> videntem. Et est locus a convertibili, quia ab uno convertibili ad alterum
+            <lb ed="#T"/> bene sequitur cum dictione exclusiva, ut tantum risibile; ergo tantum homo’. Ergo hec est
+            <lb ed="#T"/> vera ‘Sortes videt tantum omnem hominem non videntem se’. <pc type="#pilcrow"/>Item ad idem.
+            <lb ed="#T"/> Posito quod sint tantum tres homines ceci et omnes alii videant se,
+            <lb ed="#T"/> tunc iste due convertuntur ‘omnis homo cecus’ et ‘omnis homo non videns
+            <lb ed="#T"/> se’. Sed possibile est Sortem videre <add place="above-line">tantum</add> omnem hominem cecum. Ergo possibile est Sortem
+            vi<lb ed="#T"/>dere <add place="above-line">tantum</add> omnem hominem non videntem se. Ergo prima possibilis est. </p>
+
+          <p xml:id="phsyn-t3-d1e378"><pc type="#pilcrow"/>Solutio.
+            Di<lb ed="#T"/>co quod prima est falsa et impossibilis, sicut prius dictum est, quia ponit
+            contradic<lb ed="#T"/>tionem, sicut prius patuit duplici ratione. <pc type="#pilcrow"/>Ad primam autem
+            positio<lb ed="#T"/>nem dicendum quod ponit duo <seg type="correction">im<add>com</add>possibilia</seg>. Et concedo quod utrumque eorum est possibile <add place= "margin right">secundum</add>
```

Comment:
Here's a correction I already adjusted. Since the addition is "com" I've wrapped the whole word in `<seg type="corrected">` to say this word is being corrected. 

Then I've showed how it was corrected by placing the `<add>` where the addition occurs. Since there is no deletion here, there is no need for `<subst>` or `<del>`